packages/form-builder/package.json

```diff
@@ -23,8 +23,8 @@
     "@glimmer/tracking": "^1.1.2",
     "@projectcaluma/ember-core": "workspace:^",
     "@projectcaluma/ember-form": "workspace:^",
-    "codejar": "^4.2.0",
-    "ember-apollo-client": "~4.0.2",
+    "codejar": "~4.2.0",
```

Comment:
[This upstream change](https://github.com/antonmedv/codejar/pull/112) seems to break our test.

[Failing job log](https://github.com/projectcaluma/ember-caluma/actions/runs/19032255789/job/54349616957)